tests/components/deconz/test_binary_sensor.py

```diff
@@ -67,6 +67,7 @@ async def test_no_binary_sensors(hass):
     """Test that no sensors in deconz results in no sensor entities."""
     gateway = await setup_deconz_integration(hass)
     assert len(gateway.deconz_ids) == 0
+    assert len(gateway.entities[binary_sensor.DOMAIN]) == 0
```

Comment:
We don't want to assert integration details in tests of integrations. Just assert core state. That's either the state machine, the entity registry, the device registry, the service registry or the config entry.

Reply:
I'll remove them

Reply:
Fixed in #40640